pkg/agent/controller/egress/egress_controller.go

```diff
@@ -542,6 +561,38 @@ func (c *EgressController) unrealizeEgressIP(egressName, egressIP string) error
 	return nil
 }
 
+func (c *EgressController) uninstallEgressQoSMeter(meterID uint32) error {
+	// Skip the uninstallation on non-Egress Node.
+	if meterID == 0 {
+		return nil
+	}
+	c.meterTrackerMutex.Lock()
+	defer c.meterTrackerMutex.Unlock()
+	if _, exist := c.meterTracker[meterID]; exist {
+		if err := c.ofClient.UninstallEgressQoSMeter(meterID); err != nil {
+			return err
+		}
+		delete(c.meterTracker, meterID)
+	}
+	return nil
+}
+
+func (c *EgressController) installEgressQoSMeter(meterID uint32, rateLimit *crdv1b1.RateLimit) error {
+	if rateLimit == nil {
+		return nil
+	}
```

Comment:
The same check is repeated in caller and callee.

Reply:
Done.